internal/dbtest/testdata/snapshots/TestQuery-mariadb-131

```diff
@@ -0,0 +1 @@
+SELECT `id` FROM `models` AS `model` USE INDEX (`ix1, ix2`)
```

Comment:
This looks wrong - I don't think you want to quote 2 index names together ``` `ix1, ix2` ```

Reply:
Heh, passed into tests a string "`ix1, ix2`" instead "(ix1", "ix2")
:)